packages/wrangler/src/index.tsx

```diff
@@ -274,6 +275,24 @@ export function createCLIParser(argv: string[]) {
 		generateHandler
 	);
 
+	// docs
+	wrangler.command(
+		"docs",
+		"📚 Open wrangler's docs in your browser",
+		() => {},
+		async () => {
+			await printWranglerBanner();
+			const urlToOpen =
+				"https://developers.cloudflare.com/workers/wrangler/commands/";
+			logger.log(`Opening a link in your default browser: ${urlToOpen}`);
+			await openInBrowser(urlToOpen);
+			const config = readConfig(undefined, {});
```

Comment:
Why are we running this with `undefined`, `{}`?

Reply:
to make `readConfig` find it automagically

Reply:
Can we just pass it nothing? I think other places do that?